server/statistics/region_collection_test.go

```diff
@@ -25,10 +23,6 @@ import (
 	"github.com/tikv/pd/server/schedule/placement"
 )
 
-func TestStatistics(t *testing.T) {
```

Comment:
Maybe it is better to delete `func Test(t *testing.T)` in `hot_peer_cache_test.go`.